### PR TITLE
Update autocapitalize props

### PR DESCRIPTION
**Pull request by ryo-manba**
### Description

According to HTML Standard, it also accepts the following off and on keywords.
https://html.spec.whatwg.org/multipage/interaction.html#autocapitalization

This is a follow-up to the following PR.
- https://github.com/mdn/content/pull/30645

### CODE REVIEW

File: files/en-us/web/html/element/form/index.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 
   - : A nonstandard attribute used by iOS Safari that controls how textual form elements should be automatically capitalized. `autocapitalize` attributes on a form elements override it on `<form>`. Possible values:
 
-    - `none`: No automatic capitalization.
-    - `sentences` (default): Capitalize the first letter of each sentence.
+    - `none` or `off`: No automatic capitalization.
+    - `sentences` or `on` (default): Capitalize the first letter of each sentence.
```
Comment (ryo-manba) [inline files/en-us/web/html/element/form/index.md:34]:
```suggestion
    - `sentences` or `on`: Capitalize the first letter of each sentence.
```

> default
The user agent and input method should use make their own determination of whether or not to enable autocapitalization.

ref: https://html.spec.whatwg.org/multipage/interaction.html#autocapitalization

**Review by chrisdavidmills — COMMENTED** [review]
Hi there @ryo-manba , and thank you for your contribution!

I think your change is correct, but there are a few other problems with the `autocapitalize` documentation as I see it, which you could also consider adding/updating. I wrote up a quick test case at https://autocapitalize-test.glitch.me/. From my research, and from testing across different desktop and mobile browsers, I observed the following: 

- As far as I can tell, it doesn't work on desktop browsers.
- It does work across mobile browsers: I tested Firefox Android, Chrome Android, and Safari iOS. All of the values work as expected.
- The mobile browsers do differ in terms of what their default behaviors is when no `autocapitalize` is specified. On Chrome Android and Safari iOS the default behavior is `on`/`sentences`. On Firefox Android the default is `off`/`none`. This is worth documenting.
- It is in a spec and supported across multiple browsers, so you can get rid of the `{{non-standard_inline}}` macro call and the mention of non-standard in the text. Also maybe update "used by iOS Safari" to "used by mobile browsers"?
- Also, whatever changes you make here should also be made at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autocapitalize (also move it out fo the non-standard list to the standard list?) and https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea (it is covered here at all, so add it?)
- On the `<input>` page, you should document this bit from the spec: "The autocapitalize attribute never causes autocapitalization to be enabled for input elements whose type attribute is in one of the URL, Email, or Password states. (This behavior is included in the used autocapitalization hint algorithm below.)"

What do you think? Are you OK to work on these changes?

The browser compat data also needs updating to mark desktop browsers as not supporting it, and add it to element where it isn't included? This can be found at https://github.com/mdn/browser-compat-data. I appreciate that this is really another task, not part of this same one.

**Comment by ryo-manba** [comment]
@chrisdavidmills 
Thank you for the feedback! I agree with your suggestions and think they are very appropriate. I am definitely willing to work on these changes.

**Comment by ryo-manba** [comment]
@chrisdavidmills 
I have tried to fix it and would appreciate a review!

**Comment by teoli2003** [comment]
I think the easiest way to solve this is to get this PR locally (using `gh pr checkout 30724` run `npx prettier -w **/*.md` in html/element, and then commit the changes (`git commit -m "Run prettier"`) and push back using `git pull`.

More efficient than trial and error.

**Comment by chrisdavidmills** [comment]
> I think the easiest way to solve this is to get this PR locally (using `gh pr checkout 30724` run `npx prettier -w **/*.md` in html/element, and then commit the changes (`git commit -m "Run prettier"`) and push back using `git pull`.

@teoli2003 Yup, I'm going to tackle the linting errors now, by pulling it locally, don't worry. I was making some changes to the text anyway.

**Comment by chrisdavidmills** [comment]
> > I think the easiest way to solve this is to get this PR locally (using `gh pr checkout 30724` run `npx prettier -w **/*.md` in html/element, and then commit the changes (`git commit -m "Run prettier"`) and push back using `git pull`.

@teoli2003 Sorry, can you help me with this? I'm really struggling to get it to work. I don't use the github cli, so your commands won't work for me. And I can't figure out what the magical incantation is to get this to work in raw git commands.

**Comment by teoli2003** [comment]
Replace _upstream_ by _origin_ in the following if you kept the default upstream name, but this should work:
```
git fetch upstream pull/30734/head:patch-1
git checkout patch-1
```

**Comment by teoli2003** [comment]
I did a test (using `gh`), and `git push` there pushes it to ryo-manba:patch-1, so I think you should push to ryo-manba (but I have not found the git command).
Capture d’écran 2023-12-04 à 12 54 38


That's why I'm using the GitHub CLI for this, it set up everything correctly and `git push` work.

(I've added a test that I will try to remove with a git command)

**Review by chrisdavidmills — APPROVED** [review]
Thanks so much for your great work on the content @ryo-manba! It was very close to perfect; I just decided to make some fixes to the grammar before merging it.

Also a huge thanks to @teoli2003 for helping me with my git issues. I have taken note of your method, and I will look into the GitHub CLI too.

**Comment by vlakoff** [comment]
Just a nitpick, but now that this PR has been merged, could you please delete the `mdn/patch-1` branch, that was created with the Git manipulations above?

Because currently, when creating a PR using the GitHub website, contributors end up with an autogenerated branch named `patch-2`, instead of the usual `patch-1`.

**Pull request merged**